docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,107 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.ignore_filters:
+- actions: [http_request] <1>
+- categories: [database]
+  types: [creation, change, deletion] <2>
+----------------------------------------
+<1> Filters out HTTP request events
+<2> Filters out any data write events
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.ignore_filters[].actions[]` (beta)
+| List of values matched against the `event.action` field of an audit event. See <<xpack-security-audit-logging>> for a list of available events.
+
+| `xpack.security.audit.ignore_filters[].categories[]` (beta)
+| List of values matched against the `event.category` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html[ECS categorization field] for allowed values.
+
+| `xpack.security.audit.ignore_filters[].types[]` (beta)
+| List of values matched against the `event.type` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html[ECS type field] for allowed values.
+
+| `xpack.security.audit.ignore_filters[].outcomes[]` (beta)
+| List of values matched against the `event.outcome` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html[ECS outcome field] for allowed values.
+
+| `xpack.security.audit.appender` (beta)
+| Optional. Specifies where audit logs should be written to and how they should be formatted. *Default:* console
+
+2+a|
+[NOTE]
+============
+If you set `xpack.security.audit.appender` the legacy audit logger will be disabled and the new audit logger will be used instead.
+============
+|===
+
+For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.appender:
+  kind: file
+  path: /path/to/audit.log
+  layout:
+    kind: json
+----------------------------------------
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.kind` (beta)
+| Mandatory. Specifies where audit logs should be written to. Allowed values are `console` or `file` appender.
+|===
+
+[float]
+[[file-appender-settings]]
+===== File appender settings (beta)
+
+The file appender can be configured using the following settings:
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.path` (beta)
+| Mandatory. Full file path the log file should be written to.
+
+| `xpack.security.audit.appender.layout.kind` (beta)
+| Mandatory. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
+|===
+
+[float]
+[[pattern-layout-settings]]
+===== Pattern layout settings (beta)
+
+The pattern layout can be configured using the following settings:
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.layout.highlight` (beta)
+| Optional. Set to `true` to enable audit logging for security events
+
+| `xpack.security.audit.appender.layout.pattern` (beta)
+| Optional. Specifies how the log line should be formatted. *Default:* `[%date][%level][%logger]%meta %message`
```

Comment:
question: Are you aware of any docs we can link to which explains the different pattern variables that can be used?

Reply:
Unfortunately, I couldn't find the new logging settings documented in the customer facing docs. 
There's pretty good and extensive documentation in the readme file in `src/core/server/logging/README.md` but think that's it.
@restrry - Are these settings documented in the official docs too? If not are there plans to do that?

Reply:
unfortunately, `src/core/server/logging/README.md` is the only document available at the moment. we are going to declare new logging system GA after https://github.com/elastic/kibana/issues/58261